waypoints_broadcaster_node:
  ros__parameters:
    waypoints: 
      - door_wp
      - entrance
      - first_room
      - first_room_middle
      - table
      - room2
      - room2_middle
      - couch
      - room3
      - room2_entrance
      - side_room
      - clean1
    door_wp:
      position:
        x: -1.530040460241723
        y: -4.985149555638838
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: -0.9008039552261617
        w: 0.4342260174712049
    entrance:
      position:
        x: -0.8582638135044962
        y: -0.28950412631995087
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.8633233089980341
        w: 0.504651230198327
    first_room:
      position:
        x: -2.1625185216969816
        y: -0.09664011651503496
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.9775039753229956
        w: 0.21091699369121603
    first_room_middle:
      position:
        x: -2.3942810361992017
        y: 0.683519443916046
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.8627613240319747
        w: 0.5056114098342658
    table:
      position:
        x: -2.401514120552302
        y: 2.747854096062228
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: -0.2787229142814198
        w: 0.9603715619771716
    room2:
      position:
        x: -1.6495879091900802
        y: 4.356177083865586
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.23228036371641966
        w: 0.9726488742767185
    room2_middle: 
      position:
        x: -0.19440815339431894
        y: 3.987352703157562
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.9643793293115895
        w: 0.26452317326942965
    couch:
      position:
        x: -1.5387626178040776
        y: 5.091025234799736
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.7270246438172352
        w: 0.686611365535426
    room3:
      position:
        x: 1.9633151931129311
        y: 4.192730303193913
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: -0.5392175794545897
        w: 0.8421664930446552
    room2_entrance:
      position:
        x: 2.0360714433374296
        y: 2.4132548069823674
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.9457903655639737
        w: 0.3247777461686146
    side_room:
      position:
        x: -3.5812096680951893
        y: -1.93708936010797
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.8726905390046262
        w: 0.48827371742887715
    clean1:
      position:
        x: -2.3942810361992017
        y: 0.683519443916046
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.8627613240319747
        w: 0.5056114098342658

clean_waypoints:
  ros__parameters:
    start_wp: 
      - clean1
      # - room2
      # - room3
    
    clean1: ["clean_00","clean_01","clean_02", "clean_03", "clean_04", "clean_05", "clean_06"]
    # room2: ["clean_00","clean_01","clean_02"]
    # room2: ["clean_00","clean_01","clean_02"]

    clean_00:
      position:
        x: -2.303433632855246
        y: 0.12442300383605681
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.9479348505572305
        w: 0.31846431369784767

    clean_01:
      position:
        x: -2.9535238152644108
        y: 0.8438598213098171
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.9220648903394715
    clean_02:
      position:
        x: -3.565133433363479
        y: 1.6472795840489283
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.9068132230950628
        w: 0.4215326540375536
    clean_03:
      position:
        x: -2.7732889348112226
        y: 3.6495447681221926
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.44998546094731834
        w: 0.8930358811022262
    clean_04:
      position:
        x: -2.2993573783357917
        y: 3.452506783296183
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: -0.8541914242894646
        w: 0.5199586624630229
    clean_05:
      position:
        x: -3.1532396403446645
        y: 2.3654856518276373
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: -0.5729953771176881
        w: 0.8195585993702698
    clean_06:
      position:
        x: -2.25894236891745
        y: 0.8032655768894761
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: -0.4306777272437816
        w: 0.9025057868269493

      
